### ISSUE TITLE
Use a calendar *as* availability.

### ISSUE THREAD

**Issue by rikvrede** [nextcloud/calendar#6289]
### Is your feature request related to a problem? Please describe.

I work flexible hours in two different jobs, and would like to be able to book meetings with colleagues without sharing details of my personal life with them. I run four Nextcloud instances for my different jobs and family scheduling. The current implementation of "Find A Time" requires me to input a weekly unchanging schedule, which does not mesh with my ever-changing schedule.

### Describe the solution you'd like

What I would like to be able to do is create a ‘work’ calendar on my home NC server – which would show me as ‘busy’ for my family – and import that calendar into my work Nextcloud instances where it would show as my available time so colleagues could book meetings etc.

**Comment by miaulalala** [comment]
Possibly related: https://github.com/nextcloud/calendar/issues/6247

**Comment by jolentes** [comment]
That would be really great. I have the same situation to have multiple businesses with independent calendars.
If in Nextcloud Calendar on a subscription I could activate an option to consider calendar items from the subscription in my FREE/BUSY it would be really helpful.

**Comment by Tie-fighter** [comment]
I have a similar use case. I use a CRM tool, that allows booking appointments and can fetch multiple calendars to determine availability.
I would like to share my work and personal calendar(s), but without any information other than busy/available.

I imagine a more granular permission/sharing model for calendars would help a lot of people.
* Read Only
* Read Only (Availability)
* Read Write

**Comment by jvproduct** [comment]
> That would be really great. I have the same situation to have multiple businesses with independent calendars. If in Nextcloud Calendar on a subscription I could activate an option to consider calendar items from the subscription in my FREE/BUSY it would be really helpful.

Yes indeed, being able to take multiple calendars into account for checking availability is key for running a multi-client (each with their own calendar) business. Just read only would be great already.
